tests/html/Element/innerHTML.html

```diff
@@ -165,14 +173,25 @@
   });
 
   const SVG_NAMESPACE = 'http://www.w3.org/2000/svg';
+  const circleText = '<circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />';
+
+  const canSVGInnerHTML = (function() {
+    try {
+      // Edge 14 svg.innerHTML is broken even without the polyfill
+      const svg = NATIVE.createElementNS.call(document, SVG_NAMESPACE, 'svg');
+      NATIVE.innerHTML.call(svg, circleText);
+      return svg.querySelector('circle') instanceof SVGCircleElement;
+    } catch (e) {
+      return false;
+    }
```

Comment:
Is this expected to throw in some cases? If not, you don't need to try-catch, IMO.